src/components/TableData/table.css

```diff
@@ -0,0 +1,15 @@
+.ant-table-thead .ant-table-cell {
```

Comment:
Please watch the text indent

Reply:
thankyou for the fix 2313f342525894ae95ccbcbbe4d0766e8bf1e34a